test/image-sequencer.js

```diff
@@ -42,7 +42,7 @@ test('loadImages loads a DataURL image and creates a step.', function (t){
 
 if(!sequencer.options.inBrowser)
   test('loadImage loads an image from URL and creates a step. (NodeJS)', function (t){
-    sequencer.loadImage('URL','https://ccpandhare.github.io/image-sequencer/examples/red.jpg', function(){
+    sequencer.loadImage('URL','https://ccpandhare.github.io/image-sequencer/examples/images/red.jpg', function(){
```

Comment:
I like that you want to test this, but I'm concerned about making tests rely on an internet connection. Is there any other way to do this? I guess we can leave it as is for now, but you know what I mean? It just ... feels a bit wrong i guess... but I can't come up with a really really good reason why :-)